Data/Sequence.hs

```diff
@@ -13,34 +16,100 @@
 -- Maintainer  :  libraries@haskell.org
 -- Portability :  portable
 --
--- General purpose finite sequences.
--- Apart from being finite and having strict operations, sequences
--- also differ from lists in supporting a wider variety of operations
--- efficiently.
+-- = Finite sequences
+--
+-- The @'Seq' a@ type represents a finite sequence of values of
+-- type @a@.
+--
+-- Sequences generally behave very much like lists.
+--
+-- * The class instances for sequences are all based very closely on those for
```

Comment:
s/class/typeclass/ (?) Maybe its just me but its weird to see them being referred to as just "classes".

Reply:
I don't think it's weird.